### PR TITLE
add features log to leftwm-check.

### PR DESCRIPTION
**Pull request by OfekShochat**
# Description

add features log to leftwm-check - draft.

## Type of change

- [ ] Development change (no change visible to user)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation only update (no change to the factual codebase)
- [ ] This change requires a documentation update

**Note: Manual page changes must be performed in a commit, not in this PR section.**

# Checklist:

- [ ] Ran `make test-full` locally with no errors or warnings reported
  Note: To fully reproduce CI checks, you will need to run `make test-full-nix`. Usually, this is not neccesary.
- [ ] Manual page has been updated accordingly
- [ ] Wiki pages have been updated accordingly (to perform **after** merge)

### CODE REVIEW

File: leftwm/src/bin/leftwm-check.rs [hunk 1]
```diff
@@ -359,3 +365,40 @@ fn check_theme_ron(filepath: PathBuf, verbose: bool) -> Result<PathBuf> {
...
+        Err(err) => bail!("Check for feature {} failed: {:?}", name, err),
+    }
+}
+
+fn check_enabled_features() {
+    #[cfg(feature = "journald-log")]
+    check_feature("journald-log", || tracing_journald::layer()).unwrap();
+}
+
+// NOTE: this shouldnt be used. the build.rs solution is probably a lot better.
```
Comment (OfekShochat) [inline leftwm/src/bin/leftwm-check.rs:385]: this shouldnt be used

**Comment by VuiMuich** [comment]
> sure! Ill do that, and then Ill rebase (or do you squash?)

We do a squash merge usually.

**Comment by OfekShochat** [comment]
Lol, GitHub broken. Pushed, someone stopped me to talk about something heh

**Comment by OfekShochat** [comment]
Oh, I understand. They changed their clock to daylight savings...

**Comment by VuiMuich** [comment]
Sorry I got side tracked yesterday. Last commit looks good, the only tiny bit I would prefer to be enhanced a little is the formatting of the feature list would be nice: get rid of the brackets, indent like the other message lines, maybe rid the quotes. Otherwise looking good. Thanks!

**Comment by VuiMuich** [comment]
Thanks!

**Comment by OfekShochat** [comment]
sure! Ill do that, and then Ill rebase (or do you squash?)

**Pull request merged**